app/views/groups/show.html.erb

```diff
@@ -6,5 +6,17 @@
 <h2>Current Number: <%= @group.current_member_count%></h2>
 <h2>Description: <%= @group.description%></h2>
 <h2>Activity: <%= @group.activity%></h2>
+<h2>Game: <%= @game.name%></h2>
+<% if @group.current_member_count != nil %>
+  <%= form_with model: [@game, @group] do |form| %>
+    <div class="actions">
+      <div class="field">
+        <%= form.hidden_field :current_member_count, :value=>@group.current_member_count + 1 %>
+        <%= form.hidden_field :max_member_count %>
```

Comment:
This is easy to break. If two people load the page while the group is empty and both submit, the current member count will only be 1 (both users will try to set the count to 1). The increment should be done on the backend to avoid this.

Reply:
Good catch, don't know how to pass custom parameter to controller. Its fixed now but I feel my solution is still a little hacky, by introducing another permitted param. Don't know if its the best way.